libs/schemas/src/packs.ts

```diff
@@ -214,6 +214,34 @@ export const PackByTemplateIdSchema = Type.Object({
   templateId: IdSchema,
 })
 
+export const PackForPaymentSchema = Type.Intersect([
+  PackSchema,
+  Type.Object({
+    template: Type.Optional(
+      Nullable(
+        Type.Object({
+          activeBid: Type.Optional(Type.Number()),
+          allowBidExpiration: Type.Boolean(),
+          auctionUntil: Type.Optional(Type.String({ format: 'date-time' })),
+          body: Type.Optional(Type.String()),
+          collectibleTemplateIds: Type.Array(IdSchema),
+          config: PackConfigSchema,
+          image: Type.String({ format: 'uri' }),
+          onePackPerCustomer: Type.Boolean(),
+          price: Type.Number(),
+          releasedAt: Type.Optional(Type.String({ format: 'date-time' })),
+          slug: Type.String(),
+          status: Type.Enum(PackStatus),
+          subtitle: Type.Optional(Type.String()),
+          templateId: IdSchema,
+          title: Type.String(),
+          type: Type.Enum(PackType),
+        })
+      )
+    ),
+  }),
```

Comment:
Passing in an optional object with template details allows us to only send these extra details when the admin argument is used.

Reply:
Updated to use the existing `PackBaseSchema` type for `template`.